backend/qemu.pm

```diff
@@ -681,7 +680,9 @@ sub start_qemu ($self) {
     elsif ($vars->{OFW}) {
         $use_usb_kbd = $self->qemu_params_ofw;
     }
-    sp('vga', $vars->{QEMUVGA}) if $vars->{QEMUVGA};
+    $vars->{QEMUVGA} ||= 'std';
```

Comment:
Please don't set that. Some time ago I have explicitly removed the default to leave it to qemu to select a default. That fixed some problems we had